path/src/browser.ps1

```
@@ -1,33 +1,34 @@
+	if (Get-Browser) { return }
 	try {
-		if (!$browser) {
-			Test-Winget
-			winget install --id Google.Chrome -e --source winget
-			New-Item -Path "$FOUNT_DIR/data/installer" -ItemType Directory -Force | Out-Null
-			Set-Content "$FOUNT_DIR/data/installer/auto_installed_chrome" '1'
-			RefreshPath
-		}
+		Test-Winget
+		winget install --id Google.Chrome -e --source winget
 	} catch { <# ignore #> }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)